roles/redpanda_broker/templates/console/defaults.j2

```diff
@@ -0,0 +1,41 @@
+{
+  "kafka": {
+    "brokers": [
+      {% for host in advertised_ips %}
+      "{{ host }}:{{ redpanda_kafka_port }}"{% if not loop.last %},{% endif %}
+      {% endfor %}
+    ]{% if enable_tls | default(false) %},
+  "tls": {
+    "enabled": true,
+    "caFilepath": "{{ redpanda_truststore_file }}",
+    "certFilepath": "{{ redpanda_cert_file }}",
+    "keyFilepath": "{{ redpanda_key_file }}",
+    "insecureSkipTlsVerify": false
+  }
+  {% endif %}
+  },
```

Comment:
Ansible newb, but wondering if the indentation here is important? The TLS block must go under `kafka`

Reply:
Thankfully whitespace isn't a relevant consideration in JSON and other civilized languages so we don't have to worry.